spec/takeaway_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'takeaway'
+
+describe Takeaway do
+  describe '#new_order'
+
+    it "until a new_order, menu is nil" do
+      expect(subject.menu).to eq nil
+    end
+
+  let(:takeaway) { described_class.new(restaurant) }
+  let(:restaurant) { double() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. (https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)